integrations/gcp/pyproject.toml

```diff
@@ -0,0 +1,106 @@
+[tool.poetry]
+name = "gcp"
+version = "0.1.0"
+description = "A GCP ocean integration"
+authors = ["Matan Geva <matang@getport.io>"]
+
+[tool.poetry.dependencies]
+python = "^3.11"
+port_ocean = { version = "^0.5.12", extras = ["cli"] }
+google-cloud-asset = "^3.25.1"
+google-cloud-pubsub = "^2.21.1"
+google-cloud-resource-manager = "^1.12.3"
+aiostream = "^0.5.2"
```

Comment:
why? isnt it a sub package of ocean?

Reply:
Forgot to delete it once i updated to ocean 0.5.12, removed